_source/logzio_collections/_tracing-sources/span-metrics.md

```diff
@@ -726,7 +726,7 @@ Add the following parameters to the configuration file of your OpenTelemetry col
 ```yaml
 pipelines:
     traces:
-      receivers: [otlp,jaeger,opencensus,zipkin]
+      receivers: [otlp]
```

Comment:
If we reference only `otlp` in receiver pipelines we should delete all the over receivers defenitions :) ( jaeger opencensus zipkin ) 
## Now:
```
receivers:
  jaeger:
    protocols:
      thrift_compact:
        endpoint: "0.0.0.0:6831"
      thrift_binary:
        endpoint: "0.0.0.0:6832"
      grpc:
        endpoint: "0.0.0.0:14250"
      thrift_http:
        endpoint: "0.0.0.0:14268"
  opencensus:
    endpoint: "0.0.0.0:55678"
  otlp:
    protocols:
      grpc:
        endpoint: "0.0.0.0:4317"
      http:
        endpoint: "0.0.0.0:4318"
  zipkin:
    endpoint: "0.0.0.0:9411"
```
## Should be:
```
receivers:
  otlp:
    protocols:
      grpc:
        endpoint: "0.0.0.0:4317"
      http:
        endpoint: "0.0.0.0:4318"
```